src/journal-remote/journal-remote.c

```diff
@@ -30,6 +30,18 @@
 
 #define filename_escape(s) xescape((s), "/ ")
 
+#if HAVE_MICROHTTPD
+void MHDDaemonWrapper_free(MHDDaemonWrapper *d) {
```

Comment:
maybe if you touch this change this to be a proper destructor like we do it in our codebase these days, i.e. return NULL in the same type as you pass in, i.e. change to:

```c
MHDDaemonWrapper *MHDDaemonWrapper_fre(MHDDaemonWrapper *d) {
        if (!d)
                return NULL;

        …
        return mfree(d);
}
```

you get the idea